chia-blockchain/tasks/main.yml

```diff
@@ -1,4 +1,10 @@
 ---
+- name: Stop Monit Service
```

Comment:
We should probably add the same `when` condition for monit, so that we dont try to do this when monit isn't actually being installed: 

https://github.com/Chia-Network/ansible-roles/blob/main/chia-blockchain/meta/main.yml#L8